jumpscale/sals/reservation_chatflow/reservation_chatflow.py

```diff
@@ -1385,6 +1390,19 @@ def get_nodes(
         Returns:
             list of available nodes
         """
+
+        def filter_disallowed_nodes(disallowed_dict, nodes):
+            result = []
+            for node in nodes:
+                node_blocking_time = int(disallowed_dict.get(node.node_id.encode(), 0))
+                if node_blocking_time < j.data.time.now().timestamp:
+                    result.append(node)
+            return result
+
+        disallowed_dict = {}
+        if filter_blocked:
+            client = Redis()
```

Comment:
same here, you can have a redis client created somewhere and re-use it, instead of connecting every time.